libs/platform/experience/src/routes.ts

```diff
@@ -52,14 +50,19 @@ export const defaultExperienceRoutes: RouteConfig[] = [
           map((state) => (state ? '/' : !state))
         ),
   },
-  {
-    path: '/product/:sku',
-    type: RouteType.Product,
-  },
-  {
-    path: '/category/:id',
-    type: RouteType.Category,
-  },
+  ...(OryxFeatureLevel < 110
```

Comment:
How to differentiate `1.10` and `11.0`?

Reply:
https://github.com/spryker/oryx/pull/725#discussion_r1305358958

Reply:
we decided to sacrifice support for `1.10` for the time being.